### DISCUSSION TITLE
How to get the users data from the backend server?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [monkeytypegame/monkeytype#3469] [Questions] [answered]
![Screenshot 2022-08-29 at 11 38 18 AM](https://user-images.githubusercontent.com/71123337/187134034-91715c65-0058-4f8a-a566-a93e000207a2.png)
**Here??**

**Comment by Miodec** [comment]
Sorry, i don't understand the question

Just make a get request to /users or open the local database and find your user there

**Comment by nishu-murmu** [comment, marked as answer]
![Screenshot 2022-08-29 at 5 35 44 PM](https://user-images.githubusercontent.com/71123337/187197497-f274e778-ab91-407e-b282-e1f407d11f3d.png)

I'm talking about leaderboards? @Miodec

**Reply by Miodec** [reply]
Users will only appear on the leaderboards if they have more than 3 hours typed
